src/components/atoms/ImageInput/ImageInput.tsx

```diff
@@ -2,24 +2,28 @@ import React, {
   ChangeEvent,
   useCallback,
   useEffect,
+  useMemo,
   useRef,
   useState,
 } from "react";
 import { FieldError, UseFormRegister, UseFormSetValue } from "react-hook-form";
 import classNames from "classnames";
+import { Button } from "components/admin/Button";
 
-import { ACCEPTED_IMAGE_TYPES } from "settings";
+import {
+  ACCEPTED_IMAGE_TYPES,
+  DEFAULT_IMAGE_INPUT_BACKGROUND,
+  DEFAULT_IMAGE_INPUT_ROUND_BACKGOUND,
+} from "settings";
 
 import { AnyForm } from "types/utility";
 
 import { useImageInputCompression } from "hooks/useImageInputCompression";
 
-import { ImageOverlay } from "components/atoms/ImageOverlay";
-
-import { ButtonNG } from "../ButtonNG";
-
 import "./ImageInput.scss";
```

Comment:
[NOTE] this looks like a good candidate for removal.

Reply:
removed `ImageInput.scss` and moved `ImageInput` to the `admin` directory: https://github.com/sparkletown/sparkle/pull/2868/commits/ba937c968da803ade57ed5627ca35f3baeade6a0